api/app/modules/user/schemas/emergency-contact/emergency-contact.routes.js

```diff
@@ -0,0 +1,10 @@
+const EmegencyContactController = require('./emergency-contact.controller');
+
+const passport = require('passport');
+
+const isAuthenticatedMiddleware = passport.authenticate('jwt', {
+  session: false
+});
```

Comment:
Trebuie ceva de genul, nu stiu cate './../../../../../../../../' trebuiesc.
`const isAuthenticatedMiddleware = require('../../shared/middlewares/is-authenticated.middleware');`

Ce ai facut tu e reimplementarea mecanismului de jwt. trebuie sa il refolosim pe cel exportat